CONTRIBUTING.md

```diff
@@ -112,6 +112,9 @@ The `x.y.z` version numbers follow the semantics `breaking.feature.fix`:
 If applicable, breaking changes are expected to include instructions
 or scripts to perform migration on current installations.
 
+### Migrations
+
+When you you introduce a breaking change, and when you migrate a module to a new major version, you *must* provide a migration script to make it possible to upgrade from lower versions.
```

Comment:
that "must" sounds too much :)

Reply:
well, for breaking changes within a version I'd insist on the script, so in RFC speech, that's a must I think

Reply:
https://www.ietf.org/rfc/rfc2119.txt